docs/sources/developers/plugins/metadata.md

```diff
@@ -21,22 +21,28 @@ The plugin.json file is required for all plugins. When Grafana starts, it scans
-| `category`      | string                  | No       | Plugin category used on the Add data source page. Possible values are: `tsdb`, `logging`, `cloud`, `tracing`, and `sql`.                                                                                                                                                                                                                                                                                |
+| `baseUrl`       | string                  | No       |                                                                                                                                                                                                                                                                                                                                                                                                         |
+| `category`      | string                  | No       | Plugin category used on the Add data source page. Possible values are: `tsdb`, `logging`, `cloud`, `tracing`, `sql`, `enterprise` and `other`.                                                                                                                                                                                                                                                          |
 | `executable`    | string                  | No       | The first part of the file name of the backend component executable. There can be multiple executables built for different operating system and architecture. Grafana will check for executables named `<executable>_<$GOOS>_<lower case $GOARCH><.exe for Windows>`, e.g. `plugin_linux_amd64`. Combination of $GOOS and $GOARCH can be found here: https://golang.org/doc/install/source#environment. |
 | `hiddenQueries` | boolean                 | No       |                                                                                                                                                                                                                                                                                                                                                                                                         |
+| `hideFromList`  | boolean                 | No       |                                                                                                                                                                                                                                                                                                                                                                                                         |
```

Comment:
Not sure this should be advertised externally too.  It is for plugins that you can not pick explicitly like the "Getting started" panel